core/invoker/src/main/scala/whisk/core/containerpool/ContainerPool.scala

```diff
@@ -110,8 +111,15 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
           freePool = freePool - actor
           actor ! r // forwards the run request to the container
         case None =>
-          logging.error(this, "Rescheduling Run message, too many message in the pool")(r.msg.transid)
-          self ! r
+          if (r.retryCount % logRetriesInterval == 0) {
```

Comment:
does it make sense to do this time based? ie every 10seconds?

Reply:
I made that change in the replacing PR #3165